sdk/containerregistry/Azure.Containers.ContainerRegistry/src/ContainerRegistryClientOptions.cs

```diff
@@ -14,15 +14,8 @@ public class ContainerRegistryClientOptions : ClientOptions
         internal string Version { get; }
 
         /// <summary>
-        /// Gets or sets the authentication scope to use for authentication with AAD.
-        /// This defaults to the Azure Resource Manager "Azure Global" scope.  To
-        /// connect to a different cloud, set this value to "&lt;resource-id&gt;/.default",
-        /// where &lt;resource-id&gt; is one of the Resource IDs listed at
-        /// https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/services-support-managed-identities#azure-resource-manager.
-        /// For example, to connect to the Azure Germany cloud, create a client with
-        /// <see cref="AuthenticationScope"/> set to "https://management.microsoftazure.de/.default".
         /// </summary>
-        public string AuthenticationScope { get; set; } = "https://management.azure.com/.default";
+        public ContainerRegistryAudience? Audience { get; set; }
```

Comment:
I am assuming this is nullable to support default behavior in the future.

Reply:
Yes, that's correct.  In the future, if this is set we'll override the ACR scope with an ARM scope.